tests/BrewTest.php

```diff
@@ -174,9 +174,21 @@ public function test_linked_php_returns_linked_php_formula_name()
     {
         $files = Mockery::mock(Filesystem::class);
         $files->shouldReceive('isLink')->once()->with('/usr/local/bin/php')->andReturn(true);
-        $files->shouldReceive('readLink')->once()->with('/usr/local/bin/php')->andReturn('/test/path/php71/test');
+        $files->shouldReceive('readLink')->once()->with('/usr/local/bin/php')->andReturn('/test/path/php/7.3.0/test');
         swap(Filesystem::class, $files);
-        $this->assertSame('php@7.1', resolve(Brew::class)->linkedPhp());
+        $this->assertSame('php@7.3', resolve(Brew::class)->linkedPhp());
+
+        $files = Mockery::mock(Filesystem::class);
+        $files->shouldReceive('isLink')->once()->with('/usr/local/bin/php')->andReturn(true);
+        $files->shouldReceive('readLink')->once()->with('/usr/local/bin/php')->andReturn('/test/path/php/7.2.9_2/test');
+        swap(Filesystem::class, $files);
+        $this->assertSame('php@7.2', resolve(Brew::class)->linkedPhp());
+
+        $files = Mockery::mock(Filesystem::class);
+        $files->shouldReceive('isLink')->once()->with('/usr/local/bin/php')->andReturn(true);
+        $files->shouldReceive('readLink')->once()->with('/usr/local/bin/php')->andReturn('/test/path/php72/7.2.9_2/test');
+        swap(Filesystem::class, $files);
+        $this->assertSame('php@7.2', resolve(Brew::class)->linkedPhp());
 
```

Comment:
SHould we also add one for "../Cellar/php@7.2/7.2.13/bin/php" per your note above? Or am I not actually correctly following this?

Thanks!

Reply:
Good catch. Will do.

Reply:
Pushed.